### ISSUE TITLE
How can you "log in" on another computer?

### ISSUE THREAD

**Issue by kevinsimper** [beakerbrowser/fritter#14]
I want to log in on another computer, how would I do that? Without somebody else also being able to log in as me.

Wouldn't there need to a password on a private/public key that would allow only me to sign my tweets?

**Comment by pfrazee** [comment]
Hey @kevinsimper 

It's not currently possible, but we're actively working on a protocol upgrade which allows multiple devices to share control of a Dat archive, and that will solve the problem. We'll keep you updated (will leave this issue open).

**Comment by kevinsimper** [comment]
@pfrazee that sounds awesome, would it be an idea to add a help section/page to fritter to cover all the basics of this new distributed twitter that explains the small differences? 😄

**Comment by pfrazee** [comment]
Yeah that would be a good idea!

**Comment by RangerMauve** [comment]
What about using the same approach as [Rotonde](https://github.com/Rotonde/rotonde-client/issues/133) where they added a "sameAs" field to a person's profile which is an array of other dats that represent this same user.

**Comment by pfrazee** [comment]
That's also a possibility, but since multiwriter is coming, I'm inclined to wait.

**Comment by tchoi8** [comment]
I also support an option to login via multiple devices. That would be very helpful in explaining distributed network. 
Thanks for making Beakerbrowser. It's a great project!

**Comment by rjsteinert** [comment]
> What about using the same approach as [Rotonde](https://github.com/Rotonde/rotonde-client/issues/133) where they added a "sameAs" field to a person's profile which is an array of other dats that represent this same user.

I was just thinking about this problem and came to the same idea. I suppose that could get complicated in situations where we might want a user to delete a post. They would have to be on the device where they made that post... Or maybe deletes are a flag that could go in any of the archives... Either way, it sounds like multiwriter would make this simpler to solve. @pfrazee Is there a place where we can follow the progress on multiwriter support in Beaker?

**Comment by rjsteinert** [comment]
> Is there a place where we can follow the progress on multiwriter support in Beaker?

* and potentially find areas where we can lend a hand :)

**Comment by pfrazee** [comment]
@rjsteinert No, but I promise it's happening. Maf and his team are really moving quickly right now.
